src/tests/app.test.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-undef */
```

Comment:
try to a find a solution that will work in a long run as many test files comes to prevent eslint warning in test files 

try something like adding `"mocha": true` in env object of eslintrc.json file